synapse/lib/snap.py

```diff
@@ -711,8 +711,24 @@ def recurse(f, v, p, doadd=True):
         if form.isrunt:
             raise s_exc.IsRuntForm(mesg='Cannot make runt nodes.',
                                    form=form.full, prop=valu)
+
         try:
+
+            if self.core.conf.get('buid:prefetch'):
```

Comment:
should be a instance local set on snap during anit; avoids 3 derefs and a dict lookup.

Reply:
But doesn't allow a change to the config data to have a runtime effect...  worth it?

Reply:
I suppose the snap level runtime effect would be worth it.  I'll modify.